proto/net.proto

```diff
@@ -98,6 +111,18 @@ message Weights {
   // Residual tower.
   repeated Residual residual = 2;
 
+  // Embedding layer for attention body encoders.
```

Comment:
Maybe good if you add comments against new things that will only be interpreted if network type is attentionbody_with_headformat, that that is the case.

Reply:
Comments added with @borg323's PR.